src/UMAT.jl

```diff
@@ -85,19 +92,24 @@ UMAT material structure.
     MFront Abaqus interface produces specific name, e.g. `ELASTICITY_3D`.
 """
 @with_kw mutable struct UmatMaterial <: AbstractMaterial
-    drivers :: UmatDriverState = UmatDriverState()
-    ddrivers :: UmatDriverState = UmatDriverState()
-    variables :: UmatVariableState = UmatVariableState()
-    variables_new :: UmatVariableState = UmatVariableState()
-    parameters :: UmatParameterState = UmatParameterState()
-    dparameters :: UmatParameterState = UmatParameterState()
+    drivers :: UmatDriverState = UmatDriverState(NTENS=NTENS)
+    ddrivers :: UmatDriverState = UmatDriverState(NTENS=NTENS)
+    variables :: UmatVariableState = UmatVariableState(NTENS=NTENS, NSTATV=NSTATV)
+    variables_new :: UmatVariableState = UmatVariableState(NTENS=NTENS, NSTATV=NSTATV)
+    parameters :: UmatParameterState = UmatParameterState(NPROPS=NPROPS)
```

Comment:
Dear @mauro3, first of thanks for the excellent `Parameters.jl` package, I am currently learning how to use it. Sorry for pinging you directly, but I got a feeling that you might be interested in this package and maybe you can spare some minutes to guide me in the correct way. 

@IvanYashchuk made this original comment of the problem, I am trying to solve:
```
# TODO: Need to find the way how to set them and modify default state variables from user code
#       so that `reset_material!` and `update_material!` work correctly
```